packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -64,6 +64,7 @@
   "refresh_components_text": "Refresh Components",
   "force_source_retrieve_display_text": "Retrieve Source from Org",
   "force_source_retrieve_and_open_display_text": "Retrieve and Open Source",
+  "force_create_manifest": "SFDX: Generate Manifest (package.xml)",
```

Comment:
Has @sbudhirajadoc reviewed this and the other messages?

Reply:
No, she was still waiting for the vsix files, so merge will have to wait until Monday when she's back.

Reply:
@sbudhirajadoc confirmed that the string here would be better as `SFDX: Generate Manifest File`, so will be changing in the next revision.